docs/conf.py

```diff
@@ -61,6 +64,10 @@
 
 # HTML Output Options
 
-html_theme = 'qiskit'
+html_theme = 'qiskit-ecosystem'
+
```

Comment:
This makes the left sidebar show a more useful title for the site. The default includes "documentation" at the end, which is annoying.

```suggestion
html_title = f"{project} {release}"
```